utilities/os_utils.py

```diff
@@ -0,0 +1,265 @@
+import os
+from typing import Any
+
+from ocp_resources.template import Template
+
+from utilities.constants import (
+    DATA_SOURCE_NAME,
+    DV_SIZE_STR,
+    FLAVOR_STR,
+    IMAGE_NAME_STR,
+    IMAGE_PATH_STR,
+    INSTANCE_TYPE_STR,
+    LATEST_RELEASE_STR,
+    OS_STR,
+    OS_VERSION_STR,
+    PREFERENCE_STR,
+    TEMPLATE_LABELS_STR,
+    WIN_2K22,
+    WIN_2K25,
+    WIN_10,
+    WIN_11,
+    WORKLOAD_STR,
+    Images,
+)
+
+RHEL_OS_MAPPING: dict[str, dict[str, Any]] = {
+    WORKLOAD_STR: Template.Workload.SERVER,
+    FLAVOR_STR: Template.Flavor.TINY,
+    "rhel-7-8": {
+        IMAGE_NAME_STR: "RHEL7_8_IMG",
+        OS_VERSION_STR: "7.8",
+        OS_STR: "rhel7.8",
```

Comment:
not sure if it's a good idea but since rhel naming as of now is predictable and can be achieved with helper as well but could break if something change later.